### DISCUSSION TITLE
possibility skip path section

### DISCUSSION THREAD

**Discussion by blablareportblabla** [projectdiscovery/nuclei-templates#763] [Q&A] [answered]
Hi guys.

I tried to find the way to skip path section in templates.
It could be useful, when i want to process bulk of urls with different paths and just want to use matchers section.
Is it possible ?

Thanks.

**Comment by Ice3man543** [comment, marked as answer]
@c2c-pages-reports yeah, it is possible. Just use following as paths -

```yaml
paths:
  - "{{BaseURL}}"
```

This will just make request on provided path and then you can use the matchers.

**Reply by blablareportblabla** [reply]
wow. i missed it. thanks!
